CentrifyCLI.cs

```diff
@@ -120,6 +121,14 @@ class Ccli
         [Option("-apilist|--apilist", Description = "Swagger Directory and Filename", ShowInHelpText = false)]
         public string SwaggerDirectory { get; }
 
+        [Option("-br|--bootstraproles", "The roles to give access to the CLI, i.e. -br MyRole.  You may have several of these.  If not specified, default is sysadmin.", CommandOptionType.MultipleValue)]
+        public string[] BootstrapRoles { get; }
+
+        [Option("-breg|--bootstrapregexes", "The API regex to use for the CLI scope, i.e. -breg '.*'.  You may have several of these. If not specified, default is .*", CommandOptionType.MultipleValue)]

```

Comment:
.* as the default feels too permissive; I'd rather have the user has to explicitly specify all APIs and nearly everyone is going to go with the default.  Default of /core/* and /server/* maybe?

Reply:
The ability to call an API via scope does not imply the permission for that API to do anything.  The essence of bootstrap is the immediate and simple config for a human use case for sysadmins (which is the default role as well you'll note), so defaulting in this manner makes sense.  I agree however that we should update the usage wiki with various forms of this command and note the default vs more targeted behavior.